### PR TITLE
[CR] Add motor driven welding cart

### PR DESCRIPTION
**Pull request by metalyc**
### Summary
```SUMMARY: Content "Add vehicle 'Motor driven welding cart'"```

### Additional context
I think it's complete, but this is my first creation for CDDA, so feedback would be helpful if I forgot or did something wrong.

I tried to go as realistic as possible with their design, with vehicle parts already in the game, using a real motor driven welder as a template.

Modified spawns for the welding carts for my perceived realism. Examples: motor driven carts won't be indoors due to possibility of asphyxiation (basements, labs), but will be in places where power might not be always accessible or running long cables might not be feasible (construction sites, military outpost, junkyard).

### CODE REVIEW

File: data/json/vehicles/carts.json [hunk 1]
```diff
@@ -14,10 +14,7 @@
     "id": "food_cart",
     "type": "vehicle",
     "name": "Food Vendor Cart",
-    "blueprint": [
-      [ "&" ],
-      [ "H" ]
-    ],
+    "blueprint": [ [ "&" ], [ "H" ] ],
```
Comment (Night-Pryanik) [inline data/json/vehicles/carts.json]: There's no need to lint blueprints.

Comment (metalyc) [reply data/json/vehicles/carts.json]: Okay, fixed.

File: data/json/vehicles/carts.json [hunk 2]
```diff
@@ -255,6 +279,31 @@
...
+    "id": "motordriven_welding_cart",
+    "type": "vehicle",
+    "name": "Motor Driven Welding Cart",
+    "blueprint": [ "&" ],
+    "parts": [
+      {
+        "x": 0,
+        "y": 0,
+        "part": [
+          "frame_cover",
```
Comment (mlangsdorf) [inline data/json/vehicles/carts.json]: There's no welder in your welding cart.

File: data/json/vehicles/carts.json [hunk 3]
```diff
@@ -255,6 +279,31 @@
...
+          { "part": "tank_small", "fuel": "gasoline" },
+          "battery_motorbike",
+          "engine_vtwin",
+          "generator_7500w",
+          "wheel_caster",
+          "controls"
+        ]
+      }
+    ],
+    "items": [
```
Comment (mlangsdorf) [inline data/json/vehicles/carts.json]: There's no storage in your welding cart.

File: data/json/vehicles/carts.json (hunk 2 above)
Comment (metalyc) [reply data/json/vehicles/carts.json]: Well, I've re done this about 5 times now, can't believe I forgot that.

File: data/json/vehicles/carts.json (hunk 3 above)
Comment (metalyc) [reply data/json/vehicles/carts.json]: Shouldn't have any.

**Comment by ZhilkinSerg** [comment]
Something is wrong with item placement:

image

**Comment by ZhilkinSerg** [comment]
Marking as Stalled to clean up PR queue. Ping me to reopen this when you feel ready to continue working on it.

**Pull request closed without merging**